include/ddc/coordinate.hpp

```diff
@@ -15,7 +15,14 @@ using CoordinateElement = double;
  * 
  * It is tagged by its dimensions.
  */
+
 template <class... CDims>
 using Coordinate = detail::TaggedVector<CoordinateElement, CDims...>;
 
+template <class T>
+struct CoordinateOf;
+
+template <class T>
+using coordinate_of = typename CoordinateOf<T>::type;
```

Comment:
don't we use the `_t` convention for these?

Reply:
We can, I don't like it much because using struct is an internal requirement due to specialization limitations so I consider "_t" should not be visible to the user (and is a bit confusing), but we can.

Reply:
I agree with @jbigot, we tried to follow a convention. I don't think it is a good idea to introduce a new style in a MR about a new feature.